official/recommendation/ncf_keras_main.py

```diff
@@ -153,6 +153,12 @@ def run_ncf(_):
   """Run NCF training and eval with Keras."""
```

Comment:
I think we need to set `eval_batch_size` equal to `batch_size` here. (And put a `logging.warning` statement if we rewrite it.)

And then after the parse flags call, we need to set `batch_size` to `eval_batch_size`.

Reply:
Done.